ui/pages/confirm-transaction-base/confirm-transaction-base.test.js

```diff
@@ -409,6 +409,10 @@ describe('Confirm Transaction Base', () => {
         gasFeeIsCustom: true,
       },
     };
+
+    mockedStore.metamask.providerConfig.chainId = CHAIN_IDS.GOERLI;
+    mockedStore.confirmTransaction.txData.chainId = CHAIN_IDS.GOERLI;
+
```

Comment:
Why are we setting this to Goerli for unit tests?

Reply:
Not sure, I've removed the changes in this pass and the tests still passed.

Although generally, the tests in thile use GOERLI (lines 106-110)